tests/ert_tests/gui/tools/file/stdout-long-and-extra-wide

```diff
@@ -0,0 +1,49 @@
+shall be the number thou shalt count, and the number of the counting shall be
+three. Four shalt thou not count, neither count thou two, excepting that thou
+then proceed to three. Five is right out. Once the number three, being the
+third number, be reached, then lobbest thou thy Holy Hand Grenade of Antioch
+towards thy foe, who, being naughty in My sight, shall snuff it.'
+
+And Saint Attila raised the hand grenade up on high, saying, 'O Lord, bless
+this thy hand grenade, that with it thou mayst blow thine enemies to tiny bits,
+in thy mercy.' And the Lord did grin. And the people did feast upon the lambs,
+and sloths, and carp, and anchovies, and orangutans, and breakfast cereals, and
+fruit bats, and large chulapas. And the Lord spake, saying, 'First shalt thou
+take out the Holy Pin. Then shalt thou count to three, no more, no less. Three
+shall be the number thou shalt count, and the number of the counting shall be
+three. Four shalt thou not count, neither count thou two, excepting that thou
+then proceed to three. Five is right out. Once the number three, being the
+third number, be reached, then lobbest thou thy Holy Hand Grenade of Antioch
+towards thy foe, who, being naughty in My sight, shall snuff it.'
+
+And Saint Attila raised the hand grenade up on high, saying, 'O Lord, bless
+this thy hand grenade, that with it thou mayst blow thine enemies to tiny bits,
+in thy mercy.' And the Lord did grin. And the people did feast upon the lambs,
+and sloths, and carp, and anchovies, and orangutans, and breakfast cereals, and
+fruit bats, and large chulapas. And the Lord spake, saying, 'First shalt thou
+take out the Holy Pin. Then shalt thou count to three, no more, no less. Three
+shall be the number thou shalt count, and the number of the counting shall be
+three. Four shalt thou not count, neither count thou two, excepting that thou
+then proceed to three. Five is right out. Once the number three, being the
+third number, be reached, then lobbest thou thy Holy Hand Grenade of Antioch
+towards thy foe, who, being naughty in My sight, shall snuff it.'
+
+And Saint Attila raised the hand grenade up on high, saying, 'O Lord, bless
+this thy hand grenade, that with it thou mayst blow thine enemies to tiny bits,
+in thy mercy.' And the Lord did grin. And the people did feast upon the lambs,
+and sloths, and carp, and anchovies, and orangutans, and breakfast cereals, and
+fruit bats, and large chulapas. And the Lord spake, saying, 'First shalt thou
+take out the Holy Pin. Then shalt thou count to three, no more, no less. Three
+shall be the number thou shalt count, and the number of the counting shall be
+three. Four shalt thou not count, neither count thou two, excepting that thou
+then proceed to three. Five is right out. Once the number three, being the
+third number, be reached, then lobbest thou thy Holy Hand Grenade of Antioch
+towards thy foe, who, being naughty in My sight, shall snuff it.'
```

Comment:
Hei, the PR seems large than it actually is for me. The three files `stdout-ust-right`, `stdout-long` and `std-long-and-extra-wide` have the same content, just repeated to increase the size. Do you think it would have a way to avoid this repetition?

Reply:
Since we only care about the length / formatting of the files, and not about the content, and don't need to maintain the content, I thought this would be a pythonic way of solving the need for text files. introducing randomness into tests where it is not needed by the specs invites unnecessary brittleness, in my opinion...

I'll change the content of the files, to make it DRYer. it'll still seem like a long PR, but as the lengthy part is just dummy text files that don't require maintenance, I personally think that's fine. Please let me know if you strongly disagree.

Also, are there any policies / regulations / metrics regarding the number of (changed / new) lines in a PR at SCOUT or equinor that I'm not aware of? Wondering, since I got several comments on the length of this PR.

Reply:
Good point to discuss in the standup. I don't see clearly why the use of `hypothesis` would introduce problematic randomness, but do not have a strong opinion. In regards to policies, regulations and metrics, I don't know of any besides the common understanding that a PR should not be longer than it needs to be, and it should solve only one issue. But again, good to check what others think about it 🎉

Reply:
you're right, i don't think it would introduce anything dangerous or brittle in this case! BUT after lengthy discussion in the standup today, i shall merge it as is, as it's already approved.